packages/strapi-generate-model/lib/before.js

```diff
@@ -122,20 +122,6 @@ module.exports = (scope, cb) => {
     })
     .join(',\n');
 
-  // Get default connection
```

Comment:
You should just remove the JSON.parse part but allow a user to set a connection if wanted

Reply:
632432bbc5342ae7c54834fa68c37a23aa8412e6